man/setkey.Rd

```diff
@@ -55,7 +55,18 @@ haskey(x)
 \arguments{
 \item{x}{ A \code{data.table}. }
 \item{\dots}{ The columns to sort by. Do not quote the column names. If \code{\dots} is missing (i.e. \code{setkey(DT)}), all the columns are used. \code{NULL} removes the key. }
-\item{cols}{ A character vector of column names. For \code{setindexv}, this can be a \code{list} of character vectors, in which case each element will be applied as an index in turn. }
+\item{cols}{ A character vector of column names. For \code{setindexv}, this can be a \code{list} of character vectors, in which case each element will be applied as an index in turn. 
+
+# Example: Single character vector of column names
```

Comment:
thanks! please move this to the \examples{} section

Reply:
sure sir i will make a new pr for that.

Reply:
and sorry as i am new to this version control ,
I solved two issues under the same pr